src/app/shared/components/join-us/join-us.component.ts

```diff
@@ -23,12 +24,20 @@ export class JoinUsComponent implements OnInit {
 
   onSubmit() {
     this.dataService.create('', this.submitForm.value).subscribe();
-      this.msg = 'Congratulations! You are with us!';
       this.edited = false;
+      localStorage.setItem('email', JSON.stringify(this.submitForm.value));
   }
 
   invalidEmail() {
     return !this.submitForm.get('email').valid && this.submitForm.get('email').touched;
   }
 
+  checkLocal() {
+    return localStorage.getItem('email') !== null;
```

Comment:
here you can just return  localStorage.getItem('email'), if key is not exists function returns null and null is false value, in case if key exists function returns string and it is true value